failed to resolve: use of undeclared type `Errno`

Why this is error: https://github.com/containers/youki/actions/runs/4690386835/jobs/8313409869?pr=1811

```
error[E0433]: failed to resolve: use of undeclared type `Errno`
   --> crates/libcontainer/src/process/container_main_process.rs:123:23
    |
123 |             if err == Errno::ECHILD {
    |                       ^^^^^ use of undeclared type `Errno`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `libcontainer` due to previous error
warning: build failed, waiting for other jobs to finish...
Error: Process completed with exit code 101.
````

I am very confused, this code already exists in the main branch, why does the ci fail.

--- a/.github/workflows/integration_tests_validation.yaml
+++ b/.github/workflows/integration_tests_validation.yaml
@@ -5,6 +5,7 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
 
 jobs:
   changes:

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -5,6 +5,7 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
 
 jobs:
   changes:

--- a/crates/libcontainer/src/process/container_main_process.rs
+++ b/crates/libcontainer/src/process/container_main_process.rs
@@ -12,7 +12,6 @@ use nix::unistd::Pid;
 use crate::seccomp;
 #[cfg(feature = "libseccomp")]
 use nix::{
-    errno::Errno,
     sys::socket::{self, UnixAddr},
     unistd,
 };
@@ -117,15 +116,12 @@ pub fn container_main_process(container_args: &ContainerArgs) -> Result<Pid> {
             log::warn!("intermediate process killed with signal: {sig}")
         }
         Ok(_) => (),
-        Err(err) => {
+        Err(nix::errno::Errno::ECHILD) => {
             // This is safe because intermediate_process and main_process check if the process is
             // finished by piping instead of exit code.
-            if err == Errno::ECHILD {
-                log::warn!("intermediate process already reaped");
-            } else {
-                bail!("failed to wait for intermediate process: {err}");
-            }
+            log::warn!("intermediate process already reaped");
         }
+        Err(err) => bail!("failed to wait for intermediate process: {err}"),
     };
 
     Ok(init_pid)

